README.md

```diff
@@ -1,2 +1,49 @@
-# github-actions-templates
-Reusable Github composite actions and workflows
+# Kustomize validation
+
+## Summary
+
+Composite Github action that runs `kustomize build` for all folders containing a `kustomization.yaml`
+file. The action will find all `kustomization.yaml` files in the repository so there is no need to
+configure that part. The typical use case for this action is PR validation in Gitops repositories
+to make sure that the kustomization manifests are syntactically correct before merging to main.
+
+## Parameters
+
+- `version` - kustomize version to use (mandatory)
+- `checksum` - the expected kustomize checksum (mandatory)
```

Comment:
Nit: I think it's more common to use "required" over "mandatory" but maybe it's just because I read a bunch of Terraform documentation yesterday